apps/aeutils/priv/extensions/get_peer_key.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# get the peer pubkey
+CODE="{ok, PK} = aec_keys:peer_pubkey(),
+      binary_to_list(aec_base58c:encode(peer_pubkey, PK))."
+
+$(relx_nodetool eval $CODE)
```

Comment:
It looks like an explicit exit is needed? https://github.com/aeternity/epoch/blob/28f01e6b1c2923167047d7dbe503c411a0816983/apps/check_config/priv/extensions/check_config.sh#L10-L11